driver/sources/src/main/scala/org/virtuslab/ideprobe/ide/intellij/InstalledIntelliJ.scala

```diff
@@ -95,7 +97,7 @@ final class InstalledIntelliJ(val root: Path, config: DriverConfig) {
         "IDEA_VM_OPTIONS" -> vmoptions.toString,
         "IDEA_PROPERTIES" -> ideaProperties.toString,
         "IDEPROBE_DRIVER_PORT" -> server.getLocalPort.toString,
-        "IDEPROBE_OUTPUT_DIR" -> Paths.get("/tmp/ideprobe/output").toString,
+        "IDEPROBE_SCREENSHOTS_DIR" -> sys.env.getOrElse("IDEPROBE_SCREENSHOTS_DIR", paths.screenshots.toString),
```

Comment:
I didn't image this env var as some name hardcoded in probe. We already have configuration file where we can put it. Hocon supports env variables and optional config values. This place should just take `paths.screenshots` as is.
What we should change is not the test platform, but just the configuration of tests. For tests configured in HOCON, like scala extension tests and examples tests, it will be entry in configuration:
```
probe.paths.screenshots = ${?IDEPROBE_SCREENSHOTS_DIR}
```
this will overwrite existing screenshots configuration only if env var is set and leave default if it is not.
We configure probe primarily with hocon and it is quite convenient use.
ProbeDriverTests create probe programatically. We should add a method to fixture `withPaths` that will copy probe with specific paths. You will notice one problem when implementing it, that paths currently live in 2 objects: factory and fixture. I believe we should keep it in factory only, and add a getter `def` to fixture (or not even that, I am not sure if there is any need to use it directly from fixture....). Anyway the IdeProbePaths in IntelliJFactory needs to be replaced in `withPaths` method in the Fixture.

Reply:
The problem related to the screenshots directory set up explicitly in the config file with the environment variable is that we will have to remember to provide it in for every config file used in our test suites.